RELEASE_NOTES.md

```diff
@@ -51,14 +73,21 @@ For a list of all features with an announced removal target, see [Removals](intr
 
 * CSM 1.6.0 **does not support servers with NVIDIA CPUs and GPUs**. Systems with these servers should not be upgraded to CSM 1.6.0.
 
-  The January 2025 HPE HPC continuous software stack releases (CSM 1.6.0) are for HPE Cray EX systems without NVIDIA CPUs and GPUs.
-  For HPE Cray EX systems with NVIDIA CPUs and GPUs, please use the August 2024 (CSM 1.5.x) HPE HPC continuous software stack.
-  These software stacks were validated with NVIDIA HPC SDK 24.3.
-
-  The March 2025 HPE HPC continuous and extended software stack releases will be validated with NVIDIA HPC SDK 24.11.
-  The March 2025 (CSM 1.6.1) software stacks will support all HPE Cray EX systems.
-
+    * The January 2025 HPE HPC continuous software stack releases (CSM 1.6.0) are for HPE Cray EX systems without NVIDIA CPUs and GPUs.
+    * For HPE Cray EX systems with NVIDIA CPUs and GPUs, please use the August 2024 (CSM 1.5.x) HPE HPC continuous software stack.
+    * These software stacks were validated with NVIDIA HPC SDK 24.3.
+    * The March 2025 HPE HPC continuous and extended software stack releases will be validated with NVIDIA HPC SDK 24.11.
+    * The March 2025 (CSM 1.6.1) software stacks will support all HPE Cray EX systems.
 * After updating Paradise BMC firmware, the `hmcollector-poll` service will lose event subscriptions and must be restarted
     * See [Updating Foxconn Paradise Nodes with FAS](operations/firmware/FAS_Paradise.md) for details on how to do this
+* `cfs-api` pods in CLBO state during CSM install.
+    * When installing CSM 1.6, `cray-shared-kafka-kafka-` pods in the services namespace fail to come up which results in `cfs-api` pods in CLBO state.
+    * A workaround is presented in [CFS API pods in CLBO](troubleshooting/known_issues/cfs-api_pods_in_CLBO_state.md).
+* `istio-proxy` containers fail with too many open files.
+    * This may happen when any pod with `istio injection` enabled is started.
+    * A workaround is presented in [Istio-Proxy failing with too many open files](troubleshooting/known_issues/Istio-Proxy_failing_with_too_many_open_files.md)
+* IUF does not run the next stage for an activity
+    * During CSM upgrade, IUF reports that multiple sessions are in progress for an activity.
+    * A workaround is presented in [IUF does not run the next stage for an activity](troubleshooting/known_issues/iuf_unable_to_run_next_stage.md)
 
 ### Security vulnerability exceptions in CSM 1.6
```

Comment:
Any reason why this is newly introduced in 1.6? Until we provide the fix or patch we will not share OR publish all known vulnerabilities in the software.